Monika After Story/game/script-topics.rpy

```diff
@@ -1373,7 +1466,14 @@ init 5 python:
                 eventlabel="monika_whispers",
                 category=['ddlc','club members'],
                 prompt="Others still lingering",
-                random=True
+                random=True,
+                rules={
+                    "derandom_override_label": "mas_bad_derand_topic",
+                    "rerandom_callback": {
```

Comment:
since this is pure curried functions, let's just use [`renpy.partial`](https://github.com/renpy/renpy/blob/6.99.12.2003/renpy/curry.py#L68) which is a renpy provided curry implementation. I believe this is exported so it can work as `renpy.partial` as well as `renpy.curry.partial`.